src/azul/plugins/repository/tdr_anvil/__init__.py

```diff
@@ -442,6 +442,8 @@ def _retrieve_entities(self,
                            entity_type: EntityType,
                            keys: AbstractSet[Key],
                            ) -> MutableJSONs:
+        if not keys:
+            return []
```

Comment:
No bailouts, even if it makes the diff bigger. Most diff tools, including GH are pretty good at de-emphasizing whitespace changes.